### ISSUE TITLE
cnn_encoder_epoch41.pth

### ISSUE THREAD

**Issue by pygabc1** [HHTseng/video-classification#34]
HI, where I can find 
cnn_encoder_epoch41.pth ?
Thanks.

**Comment by HHTseng** [comment]
Hi, it seems that I didn't restore the results of cnn_encoder_epoch41.pth. But if you want, probably you can wait for me a few days to regain it.

**Comment by pygabc1** [comment]
> Hi, it seems that I didn't restore the results of cnn_encoder_epoch41.pth. But if you want, probably you can wait for me a few days to regain it.

Hi, HHTseng, Thank you for your comment. I realized I run wrong script. Now I am training and I have the error [Singleton array array(0, dtype=int64) cannot be considered a valid collection]
![0](https://user-images.githubusercontent.com/31337772/96035100-29efe380-0e28-11eb-88a0-84871086b4ff.png)
When I debugging your code, I saw label is generated by 
![1](https://user-images.githubusercontent.com/31337772/96035222-560b6480-0e28-11eb-8c7f-f56d4aa6b6c8.png)
How to settle it? Thanks.

**Comment by pygabc1** [comment]
> Hi, it seems that I didn't restore the results of cnn_encoder_epoch41.pth. But if you want, probably you can wait for me a few days to regain it.

Hi, HHTseng, Thank you for your reply. I realized I have to train. When I am training, I have the error [TypeError: Singleton array array(0) cannot be considered a valid collection.] in the following line.
![0](https://user-images.githubusercontent.com/31337772/96038995-9ae5ca00-0e2d-11eb-83f1-68463799da2e.png)
When I debug your code,  the label were generated in 
![1](https://user-images.githubusercontent.com/31337772/96039047-b355e480-0e2d-11eb-8160-469c67f80624.png)
How to settle it? I used just small data, which are v_ApplyEyeMakeup_g01_c01, ~~02, ~~03. Thanks.
